readme.md

```diff
@@ -156,34 +156,34 @@ lapin.respond( 'v1.users.findAll', function ( message, respond ) {
 ```
 Please refer to [JSEND](http://labs.omniti.com/labs/jsend) for standard reply attributes
 
-## Attaching Logs
-#### Logging through a service
-When attaching a logger through a service
+***Response with Validation using Joi***
 ```javascript
+// Responder
+lapin.respond( {
+    'messageType' : 'v1.users.findAll',
+    'validate'    : {
+        Joi.object().keys( {
```

Comment:
This doesn't seem right. Is this supposed to be something like `payload: Joi.object().keys( {...`?